### PR TITLE
Mhr-Frozen-Sog-Transfers

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#15732

*Description of changes:*
* Implements check and formatting for Frozen Mhr Statuses (forced Sale of gift flow)
* Various effects in SoG Transfer due to Frozen Status
* Table UX updates
* Enum name refactors
* Upgraded Typescript, linter and parser - Fixed additional misc items.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1304-opdrskyn.web.app

File: ppr-ui/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ppr-ui",
-  "version": "1.1.4",
+  "version": "1.1.5",
```
Comment (cameron-eyds) [inline ppr-ui/package.json:3]: * The change differential is mostly package-lock items, don't get scared off haha
* Hide white space changes for easier reviewing.

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1304-opdrskyn.web.app

**Review by doug-lovett — APPROVED** [review]
Looks great, lots of minor refactoring.

**Review by dimak1 — APPROVED** [review]
Nice work!

**Pull request merged**